README.md

```diff
@@ -22,7 +22,7 @@ Or manually update `require-dev` block of `composer.json` and run `composer upda
 ```json
 {
     "require-dev": {
-        "dragon-code/benchmark": "^2.0"
+        "dragon-code/benchmark": "^2.5"
```

Comment:
There is no need to specify a minor version in the description, as specifying `^2.0` will cause the composer to install the most recent version from the second series.